examples/md-flexible/src/configuration/MDFlexConfig.h

```diff
@@ -553,6 +561,35 @@ class MDFlexConfig {
    * valueOffset used for cli-output alignment
    */
   static constexpr size_t valueOffset{33};
+
+ private:
+  /**
+   * Stores the particles generated based on the provided configuration file
+   * These particles can be added to the respective autopas container,
+   * but have to be converted to the respective particle type, first.
+   */
+  std::vector<ParticleAttributes> _particles;
+
+  /**
+   * Stores the physical properties of the particles used in the an MDFlexSimulation
+   */
+  std::shared_ptr<ParticlePropertiesLibraryType> _particlePropertiesLibrary;
```

Comment:
It does not work, because std::unique_ptr deletes the copy constructor which is why we cannot create a getter for the _particlePropertiesLibrary, which we currently need.